Mathlib/CategoryTheory/Sites/Coherent/RegularSheaves.lean

```diff
@@ -97,21 +238,21 @@ lemma isSheafFor_regular_of_projective {X : C} (S : Presieve X) [S.regular] [Pro
 
 lemma EqualizerCondition.isSheaf_iff (F : Cᵒᵖ ⥤ Type*)
     [∀ ⦃X Y : C⦄ (π : X ⟶ Y) [EffectiveEpi π], HasPullback π π] [Preregular C] :
-    Presieve.IsSheaf (regularCoverage C).toGrothendieck F ↔ EqualizerCondition F := by
-  rw [Presieve.isSheaf_coverage]
+    Presieve.IsSheaf (regularTopology C) F ↔ EqualizerCondition F := by
```

Comment:
As commented above, I am not very much in favour of this renaming.

Reply:
Resolving to keep the discussion about the renaming on the first comment